### PR TITLE
[ci-app] CI Visibility – Add `cypress` Instructions

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
Add `cypress` instructions.

### Motivation
Allow users to start using the new instrumentation. 

### Preview

https://docs-staging.datadoghq.com/juan-fernandez/ci-app-add-cypress-instrumentation/continuous_integration/setup_tests/javascript/?tab=cypress

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_tests/javascript.md [hunk 1]
```diff
@@ -123,6 +125,43 @@ Run your tests as you normally do, specifying the environment where test are bei
...
+{{% tab "Cypress" %}}
+
+1. Set [`pluginsFile`][1] to `"dd-trace/cypress/plugin"`, for example through [`cypress.json`][2]:
+{{< code-block lang="json" filename="cypress.json" >}}
+{
+  "pluginsFile": "dd-trace/cypress/plugin"
+}
+{{< /code-block >}}
+
+If you've already defined a `pluginsFile`, you can still initialise the instrumentation with:
```
Comment (cg14823) [inline content/en/continuous_integration/setup_tests/javascript.md:139]: Not sure what Data Dog uses but `initialise` is the British spelling, the American spelling would be `initialize`

**Review by cg14823 — APPROVED** [review]

File: content/en/continuous_integration/setup_tests/javascript.md (hunk 1 above)
Comment (juan-fernandez) [reply content/en/continuous_integration/setup_tests/javascript.md:139]: yeah good catch. I always wonder myself 😆 . They seem to use the American spelling 👍 . Thanks!

**Review by apigirl — APPROVED** [review]
docs look good

**Comment by juan-fernandez** [comment]
> docs look good

feel free to merge whenever :-) (I can't myself)

**Pull request merged**